spring-xd-related-work.tex

```diff
@@ -49,15 +49,65 @@ \subsection{Spring XD and Storm}
 \item Building upon the functional stream processing model, users have the option
 to choose from Reactor\cite{reactor}, Spark Streaming or RxJava APIs to build
 complex data centric applications.
+\item As an alternative to linear streaming data pipelines, Spring XD provides 
+support for directed ayclic graphs (DAG)\cite{dag} through named channels to 
```

Comment:
How about:

"\item As an alternative to linear streaming data pipelines, Spring XD provides 
support for directed graphs through named channels to combine multiple flows 
upstream or downstream from the message channel. The behavior of the channel 
can be either queue-based on topic-based. However, Storm accepts Directed 
Acyclic Graph (DAG)\cite{dag} of operations that represent streaming 
computations.\\*"